app/static/dayview.css

```diff
@@ -138,29 +139,35 @@ body {
 }
 
 .event-btn {
-  text-align: center;
-  color: #fff;
-  background-color:var(--primary);
-  border-radius: 0.75rem;
-  border: none;
-  width: 3rem;
-  height: 3rem;
-  line-height: 1;
-  font-size: var(--text_xl);
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  position: absolute;
-  right: 0.5rem;
-  bottom: 0.5rem;
+    text-align: center;
+    color: #fff;
+    background-color: var(--primary);
+    border-radius: 0.75rem;
+    border: none;
+    width: 3rem;
+    height: 3rem;
+    line-height: 1;
+    font-size: var(--text_xl);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: absolute;
+    right: 0.5rem;
+    bottom: 0.5rem;
 }
 
 .plus_image {
-  width: 3rem;
-  height: 3rem;
+    width: 3rem;
+    height: 3rem;
 }
 
 .pb-1 {
-  width: 1.2rem;
-  height: 1.2rem;
+    width: 1.2rem;
+    height: 1.2rem;
+}
+
+.deleted_event {
```

Comment:
names should be lowercase and separated by "-".

Reply:
Thanks